tests/unit/components/ObsEdit/ObsEdit.test.js

```diff
@@ -48,6 +49,7 @@ const mockObservation = factory( "LocalObservation", {
 
 describe( "ObsEdit", () => {
   beforeEach( ( ) => {
+    jest.spyOn( useCurrentUser, "default" ).mockImplementation( ( ) => mockUser );
```

Comment:
Why not use a `jest.mock` here? `useCurrentUser` has no other exports than default and we do not update this mock anywhere else in this file.